CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.4.1 (2019-02-21)
+
+- Fixed radio and checkbox labels to align with chexboxes and radio buttons using a top baseline. [#50](https://github.com/blackbaud/skyux-theme/pull/50)
```

Comment:
typo in second checkbox mention "chexboxes"

Reply:
Fixed